### ISSUE TITLE
План счетов 2012 внести в эталон

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2881]
Originally reported on Google Code with ID 2881

```
Пора бы ...
Вот ссылка "G:\User\Stas\План счетов 2012.doc" 
```

Reported by `stasgm` on 2012-09-17 07:54:00

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-09-18 13:10:31
- **Labels added**: Accountancy, Etalon

**Comment by gsbelarus** [comment]
```
К этому плану счетов с 2013 года вступили в силу небольшие изменения http://normativka.by/news/show/9139/
```

Reported by `danilchyk` on 2013-05-21 07:20:58

**Comment by gsbelarus** [comment]
Reported by `danilchyk` on 2013-05-21 07:21:20

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2014-12-17 13:27:15
- **Status changed**: `Duplicate`
- **Merged into**: #3504
